release.sh

```diff
@@ -43,7 +70,12 @@ make
 # Phase 2: Modifying and pushing
 
 # Version number
-sed -i "s,^VERSION=.*$,VERSION=\'$VERSION\',g" setup.py
+# Avoid sed -i to be mac compatible
```

Comment:
I always forget that I install the GNU coreutils on my mac without the g prefix!